### PR TITLE
DDCYLS-2183: get the dstReference from session

### PR DESCRIPTION
**Pull request by ashwinirudrappa**

### CODE REVIEW

File: app/uk/gov/hmrc/digitalservicestaxstub/controllers/TaxEnrolmentCallbackController.scala [hunk 1]
```diff
@@ -73,6 +73,7 @@ class TaxEnrolmentCallbackController @Inject() (
       }
 
   def trigger(seed: String): Action[AnyContent] = Action.async { implicit request =>
+    println("====trigger====================" + seed)
```
Comment (chandraAndem) [inline app/uk/gov/hmrc/digitalservicestaxstub/controllers/TaxEnrolmentCallbackController.scala]: Println needs removing

**Review by chandraAndem — CHANGES_REQUESTED** [review]
Println needs removing, other than that everything looks good.
Will approve PR's after manual testing.

File: app/uk/gov/hmrc/digitalservicestaxstub/controllers/TaxEnrolmentCallbackController.scala (hunk 1 above)
Comment (ashwinirudrappa) [reply app/uk/gov/hmrc/digitalservicestaxstub/controllers/TaxEnrolmentCallbackController.scala]: > Println needs removing

Its already removed

**Comment by ashwinirudrappa** [comment]
> Println needs removing, other than that everything looks good. Will approve PR's after manual testing.

Its already removed

**Review by chandraAndem — APPROVED** [review]

**Pull request merged**